src/main/java/net/mcreator/blockly/java/blocks/ReturnBlock.java

```diff
@@ -25,31 +25,26 @@
 import net.mcreator.generator.template.TemplateGeneratorException;
 import net.mcreator.util.XMLUtil;
 import net.mcreator.workspace.elements.VariableElementType;
+import net.mcreator.workspace.elements.VariableElementTypeLoader;
+import org.apache.commons.lang3.StringUtils;
 import org.w3c.dom.Element;
 
-import java.util.HashMap;
-import java.util.Map;
+import java.util.*;
 
 public class ReturnBlock implements IBlockGenerator {
+	private final String[] names;
 
-	@Override public void generateBlock(BlocklyToCode master, Element block) throws TemplateGeneratorException {
-		String type = block.getAttribute("type");
-		VariableElementType returnType = null;
-
-		switch (type) {
-		case "return_logic":
-			returnType = VariableElementType.LOGIC;
-			break;
-		case "return_number":
-			returnType = VariableElementType.NUMBER;
-			break;
-		case "return_text":
-			returnType = VariableElementType.STRING;
-			break;
-		case "return_itemstack":
-			returnType = VariableElementType.ITEMSTACK;
-			break;
+	public ReturnBlock() {
+		Set<String> temp = new HashSet<>();

```

Comment:
You should use stream with .map and .collect and array collector here for cleaner code. Applies to all IBlockGenerators